tests/console/verify_bcache_partitioning.pm

```diff
@@ -0,0 +1,55 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: Validate bcache in writethrough
+# The backing device is /home and it is attached to the caching device which is the second
+# disk. Usually the caching device is a fast device (ssd). In this scenario a rotational device is used.
+# This doesnt give any significant performance advantages but it is better to test bcache with this setup
+# than use bcache without caching(which yast provides a option if we want to do so)
+# Scenarios covered:
+# - Verify that certain values are set correctly in a bcache setup after installation.
+# - Write operation in a backing device can be performed and validate that cache is working watching the cache hits.
+# Maintainer: yiannis bonatakis <ybonatakis@suse.com>
+
+use strict;
+use warnings;
+use base "opensusebasetest";
+use testapi;
+use scheduler 'get_test_suite_data';
+use Test::Assert 'assert_true';
+
+sub run {
+
+    my $test_data  = get_test_suite_data();
+    my $cachingdev = $test_data->{device};
+    select_console 'root-console';
+    # assert the registered backing dev
+    assert_script_run "cat /sys/block/bcache0/bcache/backing_dev_name | grep $test_data->{backingdev}";
+    # assert bcache is running
+    assert_script_run "cat /sys/block/bcache0/bcache/running | grep 1";
+    # assert bcache is setup and is not cached dirty data
+    assert_script_run "cat /sys/block/bcache0/bcache/state | grep clean";
+
+    record_info("bcache info", "Show info of bcache $cachingdev");
+    assert_script_run "bcache-super-show $cachingdev";
+    assert_script_run "cat /proc/partitions";
+
+    record_info("write operation");
+    my $hit_i = script_output "cat /sys/block/bcache0/bcache/stats_total/cache_hits";
+    record_info "$hit_i";
+    my $path_in_backingdev = "/home/bernhard/one/two/three";
+    assert_script_run "mkdir -p $path_in_backingdev";
+    assert_script_run "touch ${path_in_backingdev}/data";
+    assert_script_run "dd bs=1024 count=1000000 < /dev/random > ${path_in_backingdev}/data";
+
```

Comment:
Looks good, but I would suggest an additional checksum test here. So instead of some random data, write something fixed, then verify its checksum.

Reply:
will it test bcache? The dd is used only to cause some write cache operation. We can check cache-miss and others  examining the files sys/block/bcache0/bcache/ and/or bcache-super-show

Reply:
I guess to exercise the tooling for bcache here should suffice. Now that we are using / partition in bcache I guess we are already testing implicitly.